Plugin: 原神历史卡池

**插件名称：**

原神历史卡池

**插件功能：**

原神历史卡池查询

**PyPI 项目名：**

nonebot-plugin-gshisbanner

**插件 import 包名：**

nonebot_plugin_gshisbanner

**插件项目仓库/主页链接：**

https://github.com/forchannot/nonebot-plugin-gshisbanner

**标签：**

[{"label":"t:原神","color":"#ea5252"},{"label":"卡池","color":"#52ea56"}]

# 📃 商店发布检查结果

> Plugin: 原神历史卡池

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-gshisbanner 的信息如下：
    name         : nonebot-plugin-gshisbanner 
     version      : 0.2.0                      
     description  : Nonebot2查询原神历史卡池小插件        
    
    dependencies
     - httpx &gt;=0.20.0,&lt;1.0.0
     - nonebot-adapter-onebot &gt;=2.1,&lt;3.0
     - nonebot2 &gt;=2.0.0-rc.2,&lt;3.0.0
插件 nonebot_plugin_gshisbanner 加载出错：
03-15 13:44:34 [SUCCESS] nonebot | NoneBot is initializing...
03-15 13:44:34 [INFO] nonebot | Current Env: prod
03-15 13:44:34 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_gshisbanner&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_gshisbanner&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_gshisbanner/__init__.py&quot;, line 3, in &lt;module&gt;
    from .__main__ import *  # noqa
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_gshisbanner/__main__.py&quot;, line 32, in &lt;module&gt;
    NICKNAME: str = list(DRIVER.config.nickname)[0]
IndexError: list index out of range</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: t:原神-#ea5252, 卡池-#52ea56。</li><li>✅ 项目 <a href="https://github.com/forchannot/nonebot-plugin-gshisbanner">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-gshisbanner/">nonebot-plugin-gshisbanner</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4781,5 +4781,24 @@
     "homepage": "https://github.com/noneplugin/nonebot-plugin-bracket",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_gshisbanner",
+    "project_link": "nonebot-plugin-gshisbanner",
+    "name": "原神历史卡池",
+    "desc": "原神历史卡池查询",
+    "author": "forchannot",
+    "homepage": "https://github.com/forchannot/nonebot-plugin-gshisbanner",
+    "tags": [
+      {
+        "label": "t:原神",
+        "color": "#ea5252"
+      },
+      {
+        "label": "卡池",
+        "color": "#52ea56"
+      }
+    ],
+    "is_official": false
   }
 ]

